packages/create/src/cli.ts

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+import { Command } from 'commander';
+import prompts from 'prompts';
+
+const program = new Command();
+
+program
+  .name('create-sku')
+  .description('Create a new sku project')
+  .version('0.0.0')
```

Comment:
Left this [feedback](https://github.com/seek-oss/sku/pull/1365#discussion_r2308942692) on the other PR, but if we add a `./package.json` export entry, you can import this stuff from `@sku-lib/create/package.json` instead of duplicating it.